crates/weaver_forge/templates/test/converter.md

```diff
@@ -0,0 +1,11 @@
+{%- set text = "This is a TEST" -%}
```

Comment:
Maybe put `k8s` as a word in here to see what happens.

Reply:
Done